kk/views/comment.py

```diff
@@ -83,3 +83,22 @@ def votes(self, request, **kwargs):
         comment.recache_n_votes()
         # return success
         return response.Response({'status': 'Vote has been added'}, status=status.HTTP_201_CREATED)
+
+    @detail_route(methods=['post'])
+    def unvotes(self, request, **kwargs):
```

Comment:
This should probably be `unvote` (and likewise, `votes` should be `vote`), actually.